frontend/src/lib/randomize.ts

```diff
@@ -35,6 +35,27 @@ export function shuffleWithSeed(array: Vendor[], seed: string) {
   return [...shuffledPremiumWithPictures, ...shuffledWithPictures, ...shuffledWithoutPictures];
 }
 
+export function shuffleArrayWithSeed(array: string[], seed: string) {
```

Comment:
can you reuse this function above for shuffling vendors?

Reply:
Done!